docs/targets/index.md

```diff
@@ -6,6 +6,7 @@ The following is a list of Triggermesh event destinations known as `Targets`. So
 
 * AWS
     * [AWS Lambda](./awslambda.md): Forward event payload to AWS Lambda.
+    * [AWS S3](./awss3.md): Forward event payload to AWS S3 bucket.
```

Comment:
When I click on this link, it is taking me to a 404 page. The link does not match the file name. The file name shows `s3.md `
<img width="500" alt="Screen Shot 2020-11-12 at 9 42 06 AM" src="https://user-images.githubusercontent.com/54158919/98954007-58b2b580-24cb-11eb-9b65-09e4df2904ec.png">

Reply:
Good catch. The newly created file did not match the other naming conventions. This has been fixed.